### DISCUSSION TITLE
Unable to import odf data in txt format

### DISCUSSION THREAD

**Discussion by ddyyasfh** [mtex-toolbox/mtex#1763] [Ask Anything]
Dear developer:

The odf data obtained when I draw the polar graph from the xrd data, and then MTEX prompts' could not detect file format 'when I choose to import the odf data.

Many thanks,

Best,

**Comment by lfgmorales** [comment]
Hi

What format has your data?

Cheers

L

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 11 Mar 2023, at 12:28, fire ***@***.***> wrote:

﻿

The odf data obtained when I draw the polar graph from the xrd data, and then MTEX prompts' could not detect file format 'when I choose to import the odf data

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1763>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN3Y6F5JXGUJ7CB5S5DW3ROU5ANCNFSM6AAAAAAVXNZODY>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1763", "url": "https://github.com/mtex-toolbox/mtex/discussions/1763", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by ddyyasfh** [reply]
Sorry, I forgot to upload the file


[odf.txt](https://github.com/mtex-toolbox/mtex/files/10948478/odf.txt)

**Reply by ddyyasfh** [reply]
![Uploading [MGV(L10PCF6{WC8~A9)E)F.png…]()
